scout/server/blueprints/cases/views.py

```diff
@@ -1069,7 +1089,7 @@ def mark_causative(institute_id, case_name, variant_id, partial_causative=False)
         else:
             store.mark_causative(institute_obj, case_obj, user_obj, link, variant_obj)
     elif request.form["action"] == "DELETE":
-        if eval(partial_causative):
+        if partial_causative:
```

Comment:
eval = evil

Reply:
But oh so fun: I remember the first time finding it in a language. It was like I had discovered lambda calculus. "Wait, I can actually EXECUTE the code in the variable? Hold on now.."